exercise/src/cmd/generate.rs

```diff
@@ -234,8 +228,12 @@ fn generate_tests_from_canonical_data(
         .join("tests")
         .join(format!("{}.rs", exercise_name));
 
-    let tests_content =
-        fs::read_to_string(&tests_path).expect("Failed to read existing tests content.");
+    let tests_content = utils::get_tests_content(exercise_name).unwrap_or_else(|_| {
```

Comment:
`.unwrap_or_else(|_| panic!(x))` is exactly equivalent to `.expect(x)` for all `x`.

Reply:
It is actually panic!(x, y), where y is `exercise_name`:
```rust
let tests_content = utils::get_tests_content(exercise_name).unwrap_or_else(|_| {                                                                                                      
         panic!(                                                                                                                                                                           
             "Failed to get the content of the test suite for the '{}' exercise. Aborting.",                                                                                               
             exercise_name                                                                                                                                                                 
         )                                                                                                                                                                                 
}); 
```

Reply:
Ah, I see. I still think `.expect(format!(...))` is a bit more concise, but it's no big deal either way.

Reply:
I  was actually using that approach, but then `clippy` complained about it, so I decided to be use the idiomatic approach instead:

```shell
238 |       let tests_content = utils::get_tests_content(exercise_name).expect(&format!(                                                                                                      
    |  _________________________________________________________________^                                                                                                                     
239 | |         "Failed to get the content of the test suite for the '{}' exercise. Aborting.",                                                                                               
240 | |         exercise_name                                                                                                                                                                 
241 | |     ));                                                                                                                                                                               
    | |______^ help: try this: `unwrap_or_else(|_| panic!("Failed to get the content of the test suite for the '{}' exercise. Aborting.", exercise_name))`                                    
    |                                                                                                                                                                                         
    = note: #[warn(expect_fun_call)] on by default                                                                                                                                            
    = help: for further information visit https://rust-lang-nursery.github.io/rust-clippy/v0.0.212/index.html#expect_fun_call
```

Reply:
Huh. In that case, prefer clippy over my opinions.